Catappult/appcoins-wallet-android
# [WAL-78] ABTest: Balance vs My Wallet

**What does this PR do?**

   Adds base integration of Ab test
   Adds the Balance vs My Wallet Ab test

**Database changed?**

No

**Where should the reviewer start?**

ABTestCenterRepository.kt

**How should this be manually tested?**

If you are in the My Wallets groups check if the bottom navigation 4th item shows up as Wallets (current string while the final one is not created).
If not check the Rakam logs and check if the group that is being sent is Balance

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/WAL-78

**Code Review Checklist**
